client/src/rpc_response.rs

```diff
@@ -220,9 +220,17 @@ pub struct RpcStakeActivation {
     pub inactive: u64,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct RpcTokenAmount {
+    pub amount: f64,
+    pub raw_amount: u64,
```

Comment:
We might as well start with this as a String (https://github.com/solana-labs/solana/issues/11374).   Not opposed to the churn in doing the same for the rest of the new Token RPC APIs too